### PR TITLE
Add endButtonElement.onclick function

### PR DESCRIPTION
**Pull request by Varga4x4**
Tomorrow I planning to start develop the Add number site

### CODE REVIEW

**Comment by Varga4x4** [comment]
I just realised the "end" button not working correctly on the display section, I will fix it tomorrow

File: src/index.html [hunk 1]
```diff
@@ -30,11 +30,11 @@
         </main>
 
         <footer>
-            <button class="footer-button">Favourites</button>
-            <button class="footer-button">Recents</button>
-            <button class="footer-button">Contacts</button>
-            <button class="footer-button">Keypad</button>
-            <button class="footer-button">Voicemail</button>
+            <button id="footer-button">Favourites</button>
```
Comment (grandalf6) [inline src/index.html:33]: No, this should be class

File: src/index.js [hunk 2]
```diff
@@ -103,61 +109,86 @@ const IN_CALL_BUTTONS_ARRAY = [
...
+                    const buttonCharactersElement = document.createElement('p')
+                    buttonCharactersElement.innerHTML = button.characters
+                    dialButtonElement.appendChild(buttonCharactersElement)
+                }
 
-        })
+                dialButtonPadElement.appendChild(dialButtonElement)
+            })
+        }
+        renderApp()
```
Comment (grandalf6) [inline src/index.js:193]: Missing enter here

**Review by grandalf6 — APPROVED** [review]

**Comment by grandalf6** [comment]
Something is not right:

When you press call button without any numbers, nothing happends, but after you press some number, then call, then end, then Call still works

**Pull request merged**